website/docs/r/neptune_cluster.html.markdown

```diff
@@ -0,0 +1,103 @@
+* `availability_zones` - (Optional) A list of EC2 Availability Zones that
+  instances in the Neptune cluster can be created in.
+* `backup_retention_period` - (Optional) The days to retain backups for. Default `1`
+* `preferred_backup_window` - (Optional) The daily time range during which automated backups are created if automated backups are enabled using the BackupRetentionPeriod parameter.Time in UTC
+Default: A 30-minute window selected at random from an 8-hour block of time per region. e.g. 04:00-09:00
+* `preferred_maintenance_window` - (Optional) The weekly time range during which system maintenance can occur, in (UTC) e.g. wed:04:00-wed:04:30
+* `port` - (Optional) The port on which the Neptune accepts connections. Default is `8182`.
+* `vpc_security_group_ids` - (Optional) List of VPC security groups to associate
+  with the Cluster
+* `snapshot_identifier` - (Optional) Specifies whether or not to create this cluster from a snapshot. You can use either the name or ARN when specifying a Neptune cluster snapshot, or the ARN when specifying a Neptune snapshot.
+* `storage_encrypted` - (Optional) Specifies whether the Neptune cluster is encrypted. The default is `false` if not specified.
+* `replication_source_identifier` - (Optional) ARN of a source Neptune cluster or Neptune instance if this Neptune cluster is to be created as a Read Replica.
+* `apply_immediately` - (Optional) Specifies whether any cluster modifications
+     are applied immediately, or during the next maintenance window. Default is
+     `false`.
+* `neptune_subnet_group_name` - (Optional) A Neptune subnet group to associate with this Neptune instance.
+* `neptune_cluster_parameter_group_name` - (Optional) A cluster parameter group to associate with the cluster.
+* `kms_key_arn` - (Optional) The ARN for the KMS encryption key. When specifying `kms_key_arn`, `storage_encrypted` needs to be set to true.
+* `iam_roles` - (Optional) A List of ARNs for the IAM roles to associate to the Neptune Cluster.
+* `iam_database_authentication_enabled` - (Optional) Specifies whether or mappings of AWS Identity and Access Management (IAM) accounts to database accounts is enabled.
+* `engine` - (Optional) The name of the database engine to be used for this Neptune cluster. Defaults to `neptune`.
+* `engine_version` - (Optional) The database engine version.
+* `tags` - (Optional) A mapping of tags to assign to the Neptune cluster.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The Neptune Cluster Identifier
+* `cluster_identifier` - The Neptune Cluster Identifier
+* `cluster_resource_id` - The Neptune Cluster Resource ID
+* `cluster_members` – List of Neptune Instances that are a part of this cluster
+* `availability_zones` - The availability zone of the instance
```

Comment:
The attributes byline states `In addition to all arguments above` -- we should remove all arguments from this attributes list and alphabetically sort the remainder